src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -227,10 +233,22 @@ Apply to thead to make headers not bold.
     padding-top: 0;
 }
 
+.padding-top-7 {
+    padding-top: 7px;
+}
+
```

Comment:
Again, I think we should not create classes that directly maps to a CSS rule.

I understand that there are similar classes in our existing css, but (I suspect) they actually takes two other forms and are thus not merely direct mapping to CSS rules:

1. They may be edge/special values like 0, 100%, 75/25% etc., or
2. They actually have semantic meaning (e.g., "margin after question panel" or "question form padding"), and are not for one-off use.

(1 is actually a subset of 2, for example, the `.margin-0` below actually has the semantic meaning of "no margin")

In any case, perhaps an alternative approach is to use the `.question-recommended-length` class as suggested above.